cpp/tests/rolling/rolling_test.cpp

```diff
@@ -146,13 +147,14 @@ class RollingTest : public cudf::test::BaseFixture {
                         size_type min_periods)
   {
     // test all supported aggregators
```

Comment:
Hang on, why comment all of these out? These tests are for all non-string types, and they were correct as is.